src/openzaak/components/autorisaties/admin_views.py

```diff
@@ -54,27 +55,40 @@ def get_initial_for_component(
     autorisaties: List[Autorisatie],
     spec: Optional[AutorisatieSpec] = None,
 ) -> List[Dict[str, Any]]:
-    if component not in [ComponentTypes.zrc, ComponentTypes.drc, ComponentTypes.brc]:
-        return []
+    _related_objs = {}
+    _related_objs_external = []
+
+    internal_autorisaties = []
+    for autorisatie in autorisaties:
+        try:
+            obj = get_related_object(autorisatie)
+            _related_objs[autorisatie.pk] = obj
+            internal_autorisaties.append(autorisatie)
+        except ObjectDoesNotExist:
+            type_field = COMPONENT_TO_FIELDS_MAP[component]["_autorisatie_type_field"]
+            _related_objs_external.append(getattr(autorisatie, type_field))
 
-    _related_objs = {
-        autorisatie.pk: get_related_object(autorisatie) for autorisatie in autorisaties
-    }
     related_objs = {pk: obj.id for pk, obj in _related_objs.items() if obj is not None}
 
     initial = []
 
     if component == ComponentTypes.zrc:
         zaaktype_ids = set(ZaakType.objects.values_list("id", flat=True))
 
+        _initial = {"externe_typen": _related_objs_external}
+        if _related_objs_external:
+            _initial[
+                "related_type_selection"
+            ] = RelatedTypeSelectionMethods.manual_select
+
         grouped_by_va = defaultdict(list)
-        for autorisatie in autorisaties:
+        for autorisatie in internal_autorisaties:
```

Comment:
what with the external autorisaties that have the same VA, shouldn't those end up in the same group?